conanfile.py

```diff
@@ -17,38 +18,75 @@ class WinflexbisonConan(ConanFile):
+            self.output.info("Skipping download. Using cached {}".format(dlfilepath))
+        else:
+            self.output.info("Downloading {} from {}".format(name, url))
+            tools.download(url, dlfilepath)
+        tools.check_sha256(dlfilepath, sha256)
+        tools.untargz(dlfilepath)
+
+        extracted_dir = "{}-{}".format(name, self.version)
         os.rename(extracted_dir, self._source_subfolder)
+
         # Generate license from header of a source file
-        with open("%s/%s/bison/data/glr.cc" % (self.source_folder, self._source_subfolder)) as f:
+        with open(os.path.join(self.source_folder, self._source_subfolder, "bison", "data", "skeletons", "glr.cc"), ) as f:
             content_lines = f.readlines()
         license_content = []
         for i in range(2, 16):
             license_content.append(content_lines[i][2:-1])
-        tools.save("%s/%s/LICENSE" % (self.source_folder, self._source_subfolder), "\n".join(license_content))
+        tools.save(os.path.join(self.source_folder, self._source_subfolder, "COPYING"), "\n".join(license_content))
+
+    @property
+    def _fake_package(self):
+        return os.path.join(self.build_folder, "package")
 
     def build(self):
         cmake = CMake(self)
+        cmake.definitions["CMAKE_INSTALL_PREFIX"] = self._fake_package
         cmake.configure()
-        cmake.build()
+        cmake.build(args=["--config", str(self.settings.build_type)])
 
     def package(self):
-        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
-        self.copy(pattern="data/*", dst="bin", src="{}/bison".format(self._source_subfolder), keep_path=True)
-        actual_build_path = "{}/bin/{}".format(self._source_subfolder, self.settings.build_type)
-        self.copy(pattern="*.exe", dst="bin", src=actual_build_path, keep_path=False)
-        self.copy(pattern="*.h", dst="include", src=actual_build_path, keep_path=False)
+        cmake = CMake(self)
+        with tools.chdir(self.build_folder):
+            try:
+                os.mkdir(self._fake_package)
+            except IOError:
+                pass
+            cmake.install(args=["--config", str(self.settings.build_type)])
+
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+
+        self.copy(pattern="*.exe", src=self._fake_package, dst="bin",keep_path=False)
+        self.copy(pattern="*.h", src=self._fake_package, dst="include", keep_path=True)
+        self.copy(pattern="data/*", src=self._fake_package, dst="bin", keep_path=True)
+        shutil.copy(os.path.join(self.package_folder, "bin", "win_flex.exe"), os.path.join(self.package_folder, "bin", "flex.exe"))
+        shutil.copy(os.path.join(self.package_folder, "bin", "win_bison.exe"), os.path.join(self.package_folder, "bin", "bison.exe"))
 
     def package_info(self):
-        self.env_info.PATH.append(os.path.join(self.package_folder, "bin"))
+        bindir = os.path.join(self.package_folder, "bin")
```

Comment:
templates...

Reply:
This is way more user friendly.
Templates should be changed.

Reply:
https://github.com/bincrafters/conan-templates/pull/49

Reply:
Merged into the templates